resources/views/livewire/tours-table.blade.php

```diff
@@ -0,0 +1,30 @@
+<div>
+    <div>
+        <div class="mx-1">
+            <input wire:model.debounce.300ms="search" type="text" class="appearance-none border-gray-200 rounded py-3 px-4 focus:outline-none focus:bg-white focus:border-gray-500" placeholder="Search destination...">
```

Comment:
This works well, but the biggest issue you need to fix is that when you are on a page other than Page 1 and then you search something, the results are blank because there aren't enough search results to get to the page you are on.
What should happen is when you search, it needs to reset the current page to Page 1 before displaying the results.